airbyte-integrations/connectors/source-gutendex/source_gutendex/spec.yaml

```diff
@@ -0,0 +1,59 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Gutendex Spec
+  type: object
+  additionalProperties: true
+  properties:
+    author_year_start:
+      type: string
+      description: (Optional) Defines the minimum birth year of the authors. Books by authors born prior to the start year will not be returned. Supports both positive (CE) or negative (BCE) integer values
+      pattern: ^[-]?[0-9]{1,4}$
+      examples:
+        - 2002
+        - 500
+        - -500
+        - 2020
+    author_year_end:
+      type: string
+      description: (Optional) Defines the maximum birth year of the authors. Books by authors born after the end year will not be returned. Supports both positive (CE) or negative (BCE) integer values
+      pattern: ^[-]?[0-9]{1,4}$
+      examples:
+        - 2002
+        - 500
+        - -500
+        - 2020
+    copyright:
+      type: string
+      description: (Optional) Use this to find books with a certain copyright status - true for books with existing copyrights, false for books in the public domain in the USA, or null for books with no available copyright information.
+      pattern: ^(true|false|null)$
+      examples:
+        - true
+        - false
+        - null
+    languages:
+      type: string
+      description: (Optional) Use this to find books in any of a list of languages. They must be comma-separated, two-character language codes.
+      examples:
+        - en
+        - en,fr,fi
+    search:
+      type: string
+      description: (Optional) Use this to search author names and book titles with given words. They must be separated by a space (i.e. %20 in URL-encoded format) and are case-insensitive.
+      examples:
+        - dickens%20great%20expect
+        - dickens
+    sort:
+      type: string
+      description: (Optional) Use this to sort books - ascending for Project Gutenberg ID numbers from lowest to highest, descending for IDs highest to lowest, or popular (the default) for most popular to least popular by number of downloads.
+      pattern: ^(ascending|descending|popular)$
+      examples:
+        - ascending
+        - descending
+        - popular
```

Comment:
After going through the API response, what I noticed is that it only returns a maximum of 32 books per response even though the total books are more than that), and in order to get the next set of 32 books, we can use the url provided in the `next` property of the response. 

For example if I fetch data from the endpoint `/books?topic=young`, it will return a set of 32 books, and in order to get the next 32 books in the set, I will have to make a request to `/books?page=2&topic=young` and so on.

So I will try to use the url in the `next` property to make the connector incremental and implement pagination.